core/src/main/java/org/elasticsearch/search/suggest/term/TermSuggestionBuilder.java

```diff
@@ -187,10 +273,20 @@ public TermSuggestionBuilder minWordLength(int minWordLength) {
      * value higher than 1 is specified then the number cannot be fractional.
      */
     public TermSuggestionBuilder minDocFreq(float minDocFreq) {
+        checkArgument(minDocFreq > 0.0f && (minDocFreq <= 1.0f || minDocFreq == Math.floor(minDocFreq)), "minDocFreq");
```

Comment:
A value if 0 seems to be okay here. At least there are some tests in SuggestSearchTests (unfortunately in the lang-mustache module) that use that and now fail with an exception.